_templates/component/new/Component.tsx.t

```diff
@@ -5,22 +5,19 @@ unless_exists: true
 <% PascalName = h.changeCase.pascal(name) %>
 import React from 'react';
 import classNames from 'classnames';
-import { ClassValue } from 'classnames/types';
```

Comment:
🙄 This is something like the 3rd branch in which I've pushed this change. Hopefully this is the first one to merge to master.